src/goBuild.ts

```diff
@@ -25,13 +25,17 @@ export function buildCode(buildWorkspace?: boolean) {
 
 	let documentUri = editor ? editor.document.uri : null;
 	let goConfig = vscode.workspace.getConfiguration('go', documentUri);
-	outputChannel.clear();
-	outputChannel.show();
-	outputChannel.appendLine('Building in progress...');
+
+	diagnosticsStatusBarItem.show();
+	diagnosticsStatusBarItem.text = 'Building...';
 	goBuild(documentUri, goConfig, buildWorkspace)

```

Comment:
One last thing...

What we are trying to achieve here is
- Use the statusbaritem to show progress/failure when build/lint/vet is run on demand.
- Use the outputChannel to show progress/failure when build/lint/vet is run on save.

We should make sure that the statusbaritem is hidden when running build/lint/vet in save and that the outputChannel is cleared when running build/lint/vet on demand.

Reply:
Ah, okay. That makes sense.

Reply:
Actually I meant this: https://github.com/Microsoft/vscode-go/pull/1460/commits/5598cb70b662aa992b65c20085f22a2cd59b6b94